bundle/config/mutator/resolve_variable_references.go

```diff
@@ -13,15 +14,22 @@ import (
 	"github.com/databricks/cli/libs/dyn/dynvar"
 )
 
+const maxResolutionRounds = 12
```

Comment:
Can you add a comment why 12? It might not be obvious for someone reading the code later

Reply:
Added a comment